gcp-vm/src/main.tf

```diff
@@ -0,0 +1,15 @@
+module "application" {
+  source          = "github.com/massdriver-cloud/terraform-modules//massdriver-application-gcp-vm?ref=5f3a359"
+  location        = var.location
+  container_image = "${var.container.repository}/${var.container.image}:${var.container.tag}"
+  endpoint        = var.endpoint
+  max_instances   = var.max_instances
+  machine_type    = var.machine_type
+  port            = 80
+  subnetwork      = var.subnetwork
+  health_check = {
+    port = 80
+    path = "/"
```

Comment:
are we going to make this configurable?

Reply:
I did the work in both modules to make sure these values get put in the right places. I didn't add port and path to the `massdriver.yaml` because I was thinking it would be a better user experience to keep some static values in the Terraform. Kinda like Cory's suggestion for container repositories. 

What do you think, should I add the params to the `massdriver.yaml`?